backend/src/main/java/gov/cdc/usds/simplereport/db/model/auxiliary/AskOnEntrySurvey.java

```diff
@@ -1,43 +1,35 @@
 package gov.cdc.usds.simplereport.db.model.auxiliary;
 
 import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
 import java.time.LocalDate;
 import java.util.HashMap;
 import java.util.Map;
+import lombok.Builder;
 import org.json.JSONObject;
 
 /**
  * A representation of the questions we ask on test entry, somewhat but not excessively flexibly
  * arranged to be stored and retrieved as a JSON object.
  */
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
What does this annotation do? The docs say that it ignores unrecognized properties (i.e. `there are no setters or creators that accept them`) during deserialization but it looks like all of the properties _do_ have setters?

Reply:
This tells Jackson to ignore the extra deleted fields that are in the data and not in the model. without it it crashes since the data is stored in a json blob